syntax = "proto3";
package protobuf;

service CharacterService {
    rpc ExchangeSymbol (stream Message) returns (stream Message) {}
}

message Message {
    int32 idEditor = 1;
    Symbol symbol = 2;
    bool isEraseBool = 3;
}

message Symbol {
    string character = 4;
    int32 uniqueId = 5;
    repeated int32 pos = 6;
}
